### PR TITLE
PAT iptables

### PR DESCRIPTION
**Pull request by vishpat**
Added iptable rules for PAT to underlay

### CODE REVIEW

File: ansible/roles/nuage_common/defaults/main.yml [hunk 1]
```diff
@@ -7,4 +7,6 @@ nuage_ca_crt: "{{ nuage_ca_dir }}/nuageMonCA.crt"
 nuage_ca_serial: "{{ nuage_ca_dir }}/nuageMonCA.serial.txt"
 
 nuage_master_mon_dir: /usr/share/nuage-k8s-monitor
-nuage_node_plugin_dir: /usr/share/vsp-k8s
+nuage_node_plugin_dir: /usr/share/vsp-k8
```
Comment (abhat) [inline ansible/roles/nuage_common/defaults/main.yml]: Missing the s?

File: ansible/roles/nuage_node/tasks/iptables.yaml [hunk 2]
```diff
@@ -5,7 +5,13 @@
   always_run: yes
 
 - name: IPtables | Open vxlan with iptables
-  command: /sbin/iptables -I INPUT 1 -p tcp --dport 8472 -j ACCEPT -m comment --comment "nuage-vxlan"
+  command: /sbin/iptables -I INPUT 1 -p udp --dport 4789 -j ACCEPT -m comment --comment "nuage-vxlan"
   when: "'nuage-vxlan' not in iptablesrules.stdout"
-  notify:
-    - Save iptables rules
+
+- name: Accept travel traffic from overlay to underlay
```
Comment (abhat) [inline ansible/roles/nuage_node/tasks/iptables.yaml]: Allow traffic to drop from overlay to underlay?

File: ansible/roles/nuage_common/defaults/main.yml (hunk 1 above)
Comment (vishpat) [inline ansible/roles/nuage_common/defaults/main.yml]: Yes. Will fix it.

File: ansible/roles/nuage_node/tasks/iptables.yaml (hunk 2 above)
Comment (vishpat) [inline ansible/roles/nuage_node/tasks/iptables.yaml]: Yes, will make the change.

**Pull request merged**